blueflood-core/src/main/java/com/rackspacecloud/blueflood/outputs/handlers/RollupHandler.java

```diff
@@ -91,6 +93,26 @@ public RollupHandler() {
         }
     }
 
+    private enum plotTimers {
+        SPLOT_TIMER(metricsFetchTimer),
+        MPLOT_TIMER(metricsFetchTimerMPlot);
+        private Timer timer;
+
+        private plotTimers(Timer timer) {
+            this.timer = timer;
+        }
+    }
+
+    private enum rollupsOnReadTimers {
+        RR_SPLOT_TIMER(rollupsCalcOnReadTimer),
+        RR_MPLOT_TIMER(rollupsCalcOnReadTimerMPlot);
+        private Timer timer;
+
+        private rollupsOnReadTimers (Timer timer) {
+            this.timer = timer;
+        }
+    }
+
```

Comment:
@tilogaat I think it makes the code concise, but grouping the related timers together.